presto-native-execution/presto_cpp/main/operators/tests/UnsaferowShuffleTest.cpp

```diff
@@ -318,17 +320,78 @@ TEST_F(UnsafeRowShuffleTest, endToEnd) {
   }
 }
 
+TEST_F(UnsafeRowShuffleTest, StorageBasedShuffle) {
```

Comment:
naming: StorageBasedShuffle -> storageBasedShuffle

BTW, would it make sense to rename StorageBasedShuffle to PersistentShuffle?